src/WebJobs.Extensions.RabbitMQ.csproj

```diff
@@ -9,7 +9,7 @@
     <SignAssembly>false</SignAssembly>
     <AssemblyOriginatorKeyFile>PublicKey.snk</AssemblyOriginatorKeyFile>
     <DelaySign>true</DelaySign>
-    <Version>0.2.0$(VersionSuffix)-beta</Version>
+    <Version>1.0.0</Version>

```

Comment:
This should be `0.2.0$(VersionSuffix)-beta` . `beta` is needed until we GA. `VersionSuffix` is needed to generate packages with build number right?

Reply:
We can probably discuss this more if needed but what I'm thinking is, Dev branch is pointing to a version we are working towards. In this case, when a release is cut off by merging to release branch, 1.0.0 is what goes to nuget. After cutting it off, minor version in Dev branch is updated (1.1.0, 1.2.0, etc) indicating that's the version we are working towards.

All of what I described above would be automatic of course.

Reply:
Also, if dev branch needs to generate package, we do it with [pack] and that would generate with build number. Format being major.minor.patch.build (1.0.0.1, 1.0.0.2, etc).

Upgrading any of major, minor or patch version would reset this build number back to 1

Reply:
I think we need to consistent across repos with how we version packages. @brettsam might be able to provide better feedback here.

Ideally, using build numbers in the packages directly without us maintaining a counter will help co-relate a package to build number - easy to track which build generated the package.

Regardless - do bring `-beta`  back until we are ready to GA.

Reply:
We do want to keep the version suffix. Moving to 1.0.0 makes sense, but let's indeed keep the `beta` flag in the meantime.